packages/wallets/src/utils/utils.ts

```diff
@@ -7,3 +7,8 @@ type TServerError = {
 
 export const isServerError = (error: unknown): error is TServerError =>
     typeof error === 'object' && error !== null && 'code' in error;
+
+export const defineViewportVh = () => {
+    const viewportVh = window.innerHeight * 0.01;
```

Comment:
Any chances to avoid javascript for measuring stuff and setting dimensions from it? 

Binding events which does change layout to resize might potentially have performance implications. Im some cases, resize might be triggered very often (e.g. when user drags window size on desktop, given this specific handler does change .css I can imagine it might cause layout trashing). I think this specific use case should work okay, but as a rule of thumb, better to avoid.

Reply:
**Replied in slack too:** it’s dvh css property which we can’t use (lvh, svh and dvh were added in one css update) I don’t know about other values which can get innerHeight without js. I have just one suggestion, I can delete addEventListener because we need this property for mobile devices only and I guess nobody will change the size of the screen on his phone. But we have to take in attention that when the user scrolls down navigation bar hides and innerHeight becomes bigger, but when the user scrolls up then navigation bar appears and innerHeight becomes less)) so right now I don’t have any idea how to do this just with css. I’ll google more but right now I don’t know